test/ux-capture.test.js

```diff
@@ -0,0 +1,25 @@
+(function(window) {
+  //
+  // Run in either Mocha, Karma or Browser environments
+  //
+  if (typeof window === "undefined") {
+    window = {};
+  }
+
+  // UserTiming polyfill
+  window.performance = window.performance
+    ? window.performance
+    : require("usertiming");
+
+  // set up global UX object
+  const UX = window.UX ? window.UX : require("../js/ux-capture.min.js");
+
+  // assertion library
+  const expect = global.expect ? global.expect : require("expect");
+
+  describe("UX Capture", function() {
+    it("should be available as UX global variable", function() {
+      expect(UX);
+    });
+  });
+})(typeof window !== "undefined" ? window : undefined);
```

Comment:
Jest's default environment will supply a global `window` object using `jsdom`, which will allow this test to work without injecting `window` manually.

Reply:
I was already looking at peculiar behavior of UserTiming API in this environment, hope it will not be too problematic if it needs to polyfill on top of `jsdom`'s window.